app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  def create
+    post = Post.find(params[:post_id])
+    comment = Comment.new(text: params[:text], user: current_user, post:)
+    respond_to do |format|
+      format.html do
+        if comment.save
+          comment.counter_updater
+          flash[:success] = 'New comment added'
+        else
+          flash.now[:error] = 'Error: Comment could not be saved'
+        end
+        redirect_to user_post_comments_path
+      end
+    end
+  end
+end
```

Comment:
- [ ] Please add a `new` method to create a new comment as you did in the posts controller.

![image](https://user-images.githubusercontent.com/68796232/178998275-a40a27d4-cf37-4a1d-abe3-3da63d4c471c.png)


- [ ] [Optional] Please use strong parameters in this [lesson](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.) and that's to prevent any unwanted parameters added when creating a new comment.